src/coreclr/CMakeLists.txt

```diff
@@ -39,6 +39,11 @@ if (DEFINED CLR_CMAKE_ICU_DIR)
   include_directories(${CLR_CMAKE_ICU_DIR}/include)
 endif(DEFINED CLR_CMAKE_ICU_DIR)
 
+if (CLR_CMAKE_TARGET_ARCH_WASM)
+    add_compile_options(-fwasm-exceptions)
+    add_link_options(-fwasm-exceptions -sEXIT_RUNTIME=1 -sUSE_OFFSET_CONVERTER=1)
```

Comment:
Do we need the offset converter option? Docs suggest it's a workaround for old v8

Reply:
This is our usual pattern:
```c
#if __has_builtin(__builtin_return_address)
// use __builtin_return_address
#else
// use fallback
#endif
```

(`__has_builtin` itself has a fallback to nop for old toolchains)

Reply:
src\coreclr\pal\inc\rt\palrt.h(754):#define _ReturnAddress() __builtin_return_address(0)
src\coreclr\pal\src\exception\seh.cpp(251):g_SEHProcessExceptionReturnAddress = __builtin_return_address(0);
src\coreclr\pal\src\exception\signal.cpp(879):g_InvokeActivationHandlerReturnAddress = __builtin_return_address(0);

The first one can be changed to: 

```
#ifdef HOST_WASM
#define _ReturnAddress() 0
#else
...
```

The later can be changed to use `_ReturnAddress()` macro - if they are still getting compiled in for wasm.